components/dashboards-web-component/src/viewer/components/DashboardRenderer.jsx

```diff
@@ -160,6 +185,18 @@ export default class DashboardRenderer extends Component {
             DashboardThumbnail.saveDashboardThumbnail(dashboardName, dashboardContainerId);
         }
     }
+
+    onGoldenLayoutComponentAddEvent(component) {
```

Comment:
Why do we have a componentAddEvent in viewer? We don't add components in viewer but in designer.

Reply:
It was a mistake. Renamed it to onGoldenLayoutComponentCreateEvent.